lib/gds_api/locations_api.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "base"
+require_relative "exceptions"
+
+class GdsApi::LocationsApi < GdsApi::Base
+  def local_custodian_code_for_postcode(postcode)
```

Comment:
Ah, I've just noticed these two methods are missing RDoc style comments. Please have a look at [how Account API did this](https://github.com/alphagov/gds-api-adapters/blob/19f0f2c6015f2568a202f37aa205ea863c4f47ba/lib/gds_api/account_api.rb#L11-L16) and recreating that kind of comment here.